src/main/scala/com/thenewmotion/akka/rabbitmq/ConnectionActor.scala

```diff
@@ -23,7 +23,7 @@ object ConnectionActor {
 
   def props(
     factory: ConnectionFactory,
-    reconnectionDelay: FiniteDuration = 10.seconds,
+    reconnectionDelay: FiniteDuration = 3.seconds,
```

Comment:
I can agree with you, however it does not mean it is safe to change such value in open sourced library